src/assets/data/resources.json

```diff
@@ -175,5 +175,33 @@
       "url": "https://skills.github.com/",
       "tags": ["git", "github"],
       "image": "https://user-images.githubusercontent.com/1221423/156894097-ff2d6566-7b6a-4488-950e-f4ebe990965a.svg"
+    },
+    {
+        "title": "Pro Git",
+        "image": "https://git-scm.com/images/progit2.png",
+        "description": "The guide to Git and its usage.",
+        "url": "https://git-scm.com/book/en/v2",
+        "tags": ["git", "books", "documentation", "developer"]
+    },
+    {
+        "title": "Eloquent JavaScript",
+        "image": "https://nostarch.com/sites/default/files/styles/uc_product_full/public/EJS3_front.png?itok=2qHtZpJa",
+        "description": "This is a book about JavaScript, programming, and the wonders of the digital.",
+        "url": "https://eloquentjavascript.net/",
+        "tags": ["JavaScript", "books", "developer"]
+    },
+    {
+        "title": "GoHugo",
+        "image": "https://d33wubrfki0l68.cloudfront.net/c38c7334cc3f23585738e40334284fddcaf03d5e/2e17c/images/hugo-logo-wide.svg",
```

Comment:
Kindly change the image link with a proper github hosted .png image link. Thanks.

Reply:
Done✨